src/app/tests/TestReportScheduler.cpp

```diff
@@ -495,8 +495,8 @@ class TestReportScheduler
 
         // Validates that the lowest max is selected as the common max timestamp
         NL_TEST_ASSERT(aSuite, syncScheduler.mNextMaxTimestamp == node1->GetMaxTimestamp());
-        // Validates that the highest min is selected as the common min interval
-        NL_TEST_ASSERT(aSuite, syncScheduler.mNextMinTimestamp == node2->GetMinTimestamp());
+        // Validates that the highest reportable min is selected as the common min interval (0 here)
+        NL_TEST_ASSERT(aSuite, syncScheduler.mNextMinTimestamp == node1->GetMinTimestamp());
```

Comment:
why the switch from node2 to node1?

Reply:
Because we changed the logic on the min interval behavior to only consider the nodes that are reportable. In this case, node2 is not reportable so will no longer be considered to be the next min timestamp to wake at.